### PR TITLE
Suggesting Aiven open-source organization

### PR DESCRIPTION
**Pull request by dewan-ahmed**
- [ x ] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [ x ] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [ x ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ x ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ x ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
Hi @communities-oss-reviewers pls suggest what change I can make in my PR. thanks.

File: collections/open-source-organizations/index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ items:
  - komodorio/helm-dashboard
  - devtron-labs/devtron
  - socialincome-san/public
+ - https://github.com/aiven-open
```
Comment (Claudenw) [inline collections/open-source-organizations/index.md:33]: Doesn't this need to be just ` - aiven-open` or perhaps ` - aiven-open/`

Comment (dewan-ahmed) [reply collections/open-source-organizations/index.md:33]: true. but also, from the existing examples, it seems that the expectation is a github page of the form [company]/[company].github.io

will wait to see the expected format from the maintainers.

Comment (Sailor63) [reply collections/open-source-organizations/index.md:33]: What's going on

**Comment by Tely789** [comment]
1998

File: collections/open-source-organizations/index.md (hunk 1 above)
Comment (tomthorogood) [reply collections/open-source-organizations/index.md:33]: @dewan-ahmed 👋 Thanks for your contribution!

As @Claudenw mentioned, simply `- aiven-open` is what we're looking for here. Here are some [other examples](https://github.com/github/explore/blob/main/collections/made-in-france/index.md?plain=1#L10-L14) showing direct links to organizations.

Regarding your other question about `github.io` --  The entries with the `[company].github.io` format are just linking to those companies' repos that happen to have `github.io` repos, but any repository owned by an organization would work.

I hope this helps!

**Review by tomthorogood — CHANGES_REQUESTED** [review]

**Comment by dewan-ahmed** [comment]
thanks @tomthorogood 🙏🏽

**Review by Claudenw — COMMENTED** [review]
Looks like you changed.every line.  can you rebase or something to.make this the one line change you are suggesting

**Comment by dewan-ahmed** [comment]
> Looks like you changed.every line. can you rebase or something to.make this the one line change you are suggesting

Apologies for the oversight. Fixed now.

**Review by Claudenw — COMMENTED** [review]
LGTM

**Review by underwoo16 — APPROVED** [review]

**Review by tomthorogood — APPROVED** [review]

**Comment by tomthorogood** [comment]
Thanks for your contribution, @dewan-ahmed!

**Pull request merged**